amd_openvx/openvx/ago/ago_util_hip.cpp

```diff
@@ -700,7 +700,7 @@ static int agoGpuHipDataOutputAtomicSync(AgoGraph * graph, AgoData * data) {
         vx_uint32 * pNumItems = nullptr;
 
         if (data->hip_memory) {
-            hipError_t err = hipMemcpyDtoH((void *)data->buffer, (data->hip_memory + data->gpu_buffer_offset), data->size);
+            hipError_t err = hipMemcpyDtoH((void *)data->buffer, data->hip_memory, data->size + data->gpu_buffer_offset);
```

Comment:
Why do we need to copy the extra data?

Reply:
@rrawther the extra data contains the numItems (i.e., number of detected corners for FastCorner algorithm) and needs to be copied as well to the host.